website/docs/d/mutating_webhook_configuration.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_mutating_webhook_configuration"
+description: |-
+  Mutating Webhook Configuration configures a mutating admission webhook
+---
+
+# kubernetes_mutating_webhook_configuration
```

Comment:
Do a find and replace to replace all ocurrences of `kubernetes_mutating_webhook_configuration` with `kubernetes_mutating_webhook_configuration_v1` and change the filename to reflect this too.

Reply:
Fixed 👍🏾